op-e2e/faultproofs/util.go

```diff
@@ -44,6 +44,22 @@ func WithBlobBatches() faultDisputeConfigOpts {
 	}
 }
 
+func WithLatestFork() faultDisputeConfigOpts {
```

Comment:
The previous helper also set:
```
cfg.DataAvailabilityType = batcherFlags.BlobsType
```

Just wondering if we need that?

Reply:
It's not needed. We'd like to cover programs that read from blobs and without. And the op stack is designed to support batches in calldata even for the latest fork.